src/styles/blocks/starReviewsPrice.scss

```diff
@@ -0,0 +1,77 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  width: 166px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), fixed width for blocks is not recommended. Consider using max-width for better responsiveness.